rasa_sdk/events.py

```diff
@@ -86,12 +97,24 @@ def ReminderScheduled(
 
 # noinspection PyPep8Naming
 def ReminderCancelled(
-    action_name: Text, name: Optional[Text] = None, timestamp: Optional[float] = None
+    name: Optional[Text] = None,
+    intent_name: Optional[Text] = None,
+    entities: Optional[Union[List[Dict[Text, Any]], Dict[Text, Text]]] = None,
+    timestamp: Optional[float] = None,
 ) -> EventType:
+    if intent_name and (
```

Comment:
that's duplicate. can we put it in a function and re-use it?

Reply:
Is it ok to call the function `_is_probably_action_name`? I know it's not an "intent" like `_check_if_probably_action_name`, but in this case the former is actually more clear.

Reply:
good to me :-)

Reply:
wait, `is_...` indicates that the function returns a `bool` to me :-D Maybe something like `_check_potential_action_usage`?

Reply:
It does return `bool`. I just pushed it, for clarification.

Reply:
The messages are different for `ReminderScheduled` and `ReminderCancelled`, so I only "outsourced" the condition.